crates/summit/src/queue.rs

```diff
@@ -170,29 +171,29 @@ impl Queue {
 
         let mut num_blocked = 0;
 
-        for blocked in self.0.iter().filter(|queued| queued.dependencies.contains(&task)) {
+        for blocked in self.0.iter().filter(|queued| queued.dependencies.contains(&task_id)) {
             task::block(tx, blocked.task.id, &blocker_id)
                 .await
                 .context("add task blocker")?;
 
             num_blocked += 1;
 
-            warn!(task = %blocked.task.id, blocker = %removed.task.id, "Task blocked");
+            warn!(task_id = %blocked.task.id, blocker = %removed.task.id, "Task blocked");
         }
 
         if num_blocked == 0 {
-            debug!("No dependents to block");
+            debug!(task_id = %task_id, "No dependents to block");
```

Comment:
Shorter way of saying the same thing:

```suggestion
            debug!(%task_id, "No dependents to block");
```

Reply:
Actually, this fn already uses a span via `tracing::instrument` that records the task ID. You don't have to mention it in the individual events at all here.